### PR TITLE
Target analysis

### PR DESCRIPTION
**Pull request by neelasha23**
Target analysis plot

### CODE REVIEW

File: src/sklearn_evaluation/plot/target_analysis.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        return
+
+    y_type = type_of_target(y)
+    if y_type not in ("binary", "multiclass"):
+        raise TypeError(
+            "'{}' target type not supported, only binary and multiclass".
+            format(y_type))
+
+
+def target_plot(y_train, y_test=None, labels=None, colors=None, ax=None):
```
Comment (yafimvo) [inline src/sklearn_evaluation/plot/target_analysis.py:42]: We need to track the usage of this method. We can do it with `@SKLearnEvaluationLogger.log(feature='plot')`

**Review by yafimvo — CHANGES_REQUESTED** [review]
Tested it locally and it looks good! One note, I think we should track `def target_plot` usage.

**Comment by yafimvo** [comment]
looking good to me
@idomic @neelasha23

**Comment by idomic** [comment]
@neelasha23 looking good, the only thing that's missing is documentation, did you add it besides the sample notebook? We need to add some details about this plot, when to use it, what parameters it takes, common pitfalls etc. into the right section (let's say metrics)

**Comment by neelasha23** [comment]
When to use and parameters are mentioned in the guide. I will look into common pitfalls if any
@idomic

**Comment by edublancas** [comment]
@neelasha23 for some reason, github did not trigger the CI. can you try adding a new commit and push?

```
git commit --allow-empty -m 'trigger ci'
```

**Comment by neelasha23** [comment]
Done @edublancas

**Review by edublancas — APPROVED** [review]

**Comment by edublancas** [comment]
nice work! 🚀

**Pull request merged**